### DISCUSSION TITLE
Could this be a thing

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#305] [Q&A]
Would there be any simple or concise way to make this a thing?
![gradient notes idea](https://github.com/FNF-CNE-Devs/CodenameEngine/assets/89645987/ba8bd45f-ddc0-4caf-963c-822127b0516b)

**Comment by r6915ee** [comment]
So if I'm getting this correct, the farther the hold strum is from the main notes, the more transparent it gets?

**Reply by THEkatinamicrowave** [reply]
Basically, in the vocals some of the vocals fade in volume. I want that to reflect in the hold note.

**Reply by THEkatinamicrowave** [reply]
here's an example

https://github.com/FNF-CNE-Devs/CodenameEngine/assets/89645987/3eeea3e7-e03f-4c66-babb-4b2baf12c4e3

**Reply by THEkatinamicrowave** [reply]
I made like a kinda layout thing

```
if notetype == fadestrum
       [calculate the amount of holdsprites used for the strum]
       [remove the cap texture]
       for texture in strum
               [opacity of the last sprite] - [100 / amount of holdsprites]
       }
}
```
obviously not functional code, but it serves as a decent layout
